src/health.js

```diff
@@ -16,12 +16,13 @@ router.get("/live", (_, res) => res.status(200).json({ status: "ok" }));
  * @response 200 - OK
  * @response 503 - Service Unavailable
  */
-router.get("/ready", async (req, res, next) => {
```

Comment:
why are we removing `req`?

Reply:
oh the rest all do that too I guess, idk which I like better, the linting warning is annoying, but also not a huge fan of `(_, res)` especially because we can't do `(_, _)` because `_` is an actual variable still

I think `(_res, _req)` will also silence the warnings maybe we should opt for that?

@moficodes what's your opinion?

Reply:
Will readd `req`

Reply:
Done